packages/mobile/src/components/text/text.tsx

```diff
@@ -5,18 +5,20 @@ import { styles } from './styles';
 
 type TextProps = RNTextProps & {
   children: string;
+  color?: string;
```

Comment:
There is a `style` prop for such purposes

Reply:
fixed